src/main/java/me/zeroeightsix/kami/module/modules/player/FastExp.java

```diff
@@ -0,0 +1,100 @@
+package me.zeroeightsix.kami.module.modules.player;
+
+import me.zero.alpine.listener.EventHandler;
+import me.zero.alpine.listener.Listener;
+import me.zeroeightsix.kami.event.events.PacketEvent;
+import me.zeroeightsix.kami.module.Module;
+import me.zeroeightsix.kami.module.modules.combat.Aura;
+import me.zeroeightsix.kami.setting.Setting;
+import me.zeroeightsix.kami.setting.Settings;
+import net.minecraft.init.Items;
+
+/**
+ * Created 17 October 2019 by hub
+ * Updated 21 November 2019 by hub
+ */
+@Module.Info(name = "FastExp", category = Module.Category.PLAYER, description = "Auto Switch to XP and throw fast")
+public class FastExp extends Module {
+
+    private Setting<Boolean> autoThrow = register(Settings.b("Auto Throw", true));
+    private Setting<SwitchMode> switchMode = register(Settings.e("Auto Switch", SwitchMode.ON));
```

Comment:
Why not a boolean setting?

Reply:
changed to `Setting<Boolean>` [here](https://github.com/zeroeightysix/KAMI/pull/144/commits/c57ea9eddf376f28ef179b71be984c2da6768aa6).